lib/commands/general.js

```diff
@@ -25,6 +26,16 @@ commands.keys = async function (keys) {
   return await this.bootstrap.sendAction('setText', params);
 };
 
+commands.getDeviceTime = async function() {
+  log.info('Attempting to capture android device date and time');
+  try {
+    await this.adb.shell(['-s', this.opts.udid],'date')
```

Comment:
Need to get the standard output from this call, to use in the next line. Something like we find here https://github.com/appium/appium-android-driver/blob/6e6478c6acab054b1966fdd2090412090726f3ff/lib/webview-helpers.js#L20